orion/initializer.go

```diff
@@ -275,6 +276,10 @@ func (e *errorLoggingInitializer) Init(svr Server) error {
 		notifier.SetNotifier(sentryNotifier)
 		logging.Debug(context.Background(), "sentryDSN", rToken, "env", env)
 	}
+
+	// default logger : stdlog
+	logger := stdlog.NewLogger()
+	logging.RegisterLogger(logger)
```

Comment:
@bhadreswar-ghuku Instead of registering a logger explicitly. I am thinking we should make ensure `logging` module can at least log to console without any initialization.

I created this PR for this idea https://github.com/carousell/logging/pull/1